### ISSUE TITLE
New Post Notification for Chrome Tabs does not change unless you either move the tab or refresh

### ISSUE THREAD

**Issue by ghost** [ihavenoface/4chan-x#92]
It's really annoying. I'll be watching YouTube while watching a thread in a second tab, then the **(1) /g/ - Install Gentoo** tab notification will pop up. Then I will mouse over and click on the tab and scroll down to the new content.

And then the tab will not change from **(1) /g/ - Install Gentoo** unless I refresh or move the tab.

**Comment by ihavenoface** [comment]
I can't reproduce this and it rather seems to be a problem with your browser since the script does it right with `$.on(window, 'scroll focus', Unread.scroll);`.

**Comment by Weyugioh** [comment]
I've also had to deal with this problem. I've tried to reinstall/update Chrome, disable/reinstall extensions, and nothing seems to work.
I've noticed that it started to happen since around 2.36.1. Anything higher than that gave me the problem.
After trying to figure out what it was by myself, it seemed it wouldn't change from (1) to (0) unless the post was more than a couple of words. Posts with pictures didn't seem to give me the problem, but it would happen uncommonly. Nothing concrete, but maybe there is a problem with scrolling a small amount.
